src/Xrefcheck/Scan.hs

```diff
@@ -23,11 +23,13 @@ import Data.Aeson.TH (deriveFromJSON)
 import Data.Foldable qualified as F
```

Comment:
Please go through the PR checklist.
Since this PR affects the user, we should update the changelog.